src/libfetchers/cache.hh

```diff
@@ -10,6 +10,44 @@ struct Cache
 {
     virtual ~Cache() { }
 
+    /* A cache for arbitrary Attrs -> Attrs mappings with a timestamp
+       for expiration. */
```

Comment:
This should be a doc comment on `Cache`? Also good to put `Attrs` in backticks (Doxygen understands some markdown these days).